src/Resources/config/no_driver.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="sonata.media.manager.media" class="Sonata\MediaBundle\Model\NoDriverManager"/>
+        <service id="sonata.media.manager.gallery" class="Sonata\MediaBundle\Model\NoDriverManager"/>
+        <service id="sonata.media.generator.default" class="Sonata\MediaBundle\Generator\NoDriverGenerator"/>
```

Comment:
Please mark service than can be private as such

Reply:
@greg0ire only `sonata.media.generator.default` can be private, two others must be public. I will add `public="true"` there